.github/workflows/build.yml

```
@@ -642,33 +461,32 @@ jobs:
         run: |
           df -h
 
+      - name: Replace version & specify dynamic libraries
+        shell: bash
+        run: |
+          if [[ ${{ matrix.os }} == macos-* ]]; then
+            gsed -i "s/__version__ = \"latest\"/__version__ = \"${{ env.VOICEVOX_ENGINE_VERSION }}\"/" voicevox_engine/__init__.py
+            echo 'LIBCORE_PATH=download/core/libcore.dylib' >> $GITHUB_ENV
+            echo 'LIBONNXRUNTIME_PATH=download/onnxruntime/lib/libonnxruntime.dylib' >> $GITHUB_ENV
+          else
+            sed -i "s/__version__ = \"latest\"/__version__ = \"${{ env.VOICEVOX_ENGINE_VERSION }}\"/" voicevox_engine/__init__.py
+            echo 'LIBCORE_PATH=download/core/core.dll' >> $GITHUB_ENV
+            echo 'LIBONNXRUNTIME_PATH=download/onnxruntime/lib/onnxruntime.dll' >> $GITHUB_ENV
+          fi
```

このままでも全く問題ないのですが、この環境変数はpyinstallerのみで使われるローカル変数にしちゃっても良いかも？

下のpyinstallerコマンドとくっつけてこんな感じにしちゃうとか。

```suggestion
          if [[ ${{ matrix.os }} == macos-* ]]; then
            gsed -i "s/__version__ = \"latest\"/__version__ = \"${{ env.VOICEVOX_ENGINE_VERSION }}\"/" voicevox_engine/__init__.py
            LIBCORE_PATH=download/core/libcore.dylib
            LIBONNXRUNTIME_PATH=download/onnxruntime/lib/libonnxruntime.dylib
          else
            sed -i "s/__version__ = \"latest\"/__version__ = \"${{ env.VOICEVOX_ENGINE_VERSION }}\"/" voicevox_engine/__init__.py
            LIBCORE_PATH=download/core/core.dll
            LIBONNXRUNTIME_PATH=download/onnxruntime/lib/onnxruntime.dll
          fi

          LIBCORE_PATH="$LIBCORE_PATH" \
          LIBONNXRUNTIME_PATH="$LIBONNXRUNTIME_PATH" \
          pyinstaller --noconfirm run.spec
```

---

なるほど！
ちょっとよく分かってなくて、環境変数が適用されないなーと手こずっていたところでした...
変えてみますね...!